contracts/ERC721AStorage.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
```

Comment:
Do we need to make this 0.8.4 to match ERC721AUpgradeable?

Reply:
Same with the other pragmas

Reply:
I think we can just leave it at 0.8.0.